state/application.go

```diff
@@ -356,15 +355,15 @@ func (op *DestroyApplicationOperation) destroyOps() ([]txn.Op, error) {
 			op.AddError(err)
```

Comment:
This should then be removed too, no? as well as !op.Force above...

Reply:
still see it ...

Reply:
I assumed that errors originating from `countApplicationOffersRefOp` were considered operational errors, and so could be ignored by force.